internal/pkg/print/print.go

```diff
@@ -165,9 +165,15 @@ func (p *Printer) PagerDisplay(content string) error {
 	}
 	pagerCmd := exec.Command("less", "-F", "-S", "-w")
 
+	pager, pagerExists := os.LookupEnv("PAGER")
```

Comment:
It this a commonly known env var?

Reply:
yes it is, a lot of tools uses PAGER env variable.